.prettierignore

```diff
@@ -21,14 +21,8 @@ __generated__/
 
 # crates
 crates/*/js/src/compiled
-crates/turbopack/bench.json
-crates/turbopack/tests
-crates/turbopack-ecmascript/tests/analyzer/graph
-crates/turbopack-ecmascript/tests/tree-shaker
 crates/next-transform-strip-page-exports/tests
-crates/next-transform-dynamic/tests
-crates/turbopack-tests/tests/execution/turbopack/basic/error/input/broken.js
-crates/turbopack-tests/tests/execution/**/output*
+crates/next-transform-dynamic/tes
```

Comment:
We should be able to remove this one as well

Reply:
thanks!! I dug a bit deeper and actually all of them under `# crates` seem to have been turbopack specific.  e.g. (from a checkout prior to the turbopack removal):

![Screenshot_20240801_170149](https://github.com/user-attachments/assets/662c1b9f-19a8-4444-a230-f63ed53d67c2)

see: f32cf13524f7697d34f6800057794fc323ac2918

Reply:
Makes sense, we can just nuke them all